src/hotspot/share/runtime/globals.hpp

```diff
@@ -247,6 +247,9 @@ const int ObjectAlignmentInBytes = 8;
           "page size for the environment as the maximum)")                  \
           range(0, max_uintx)                                               \
                                                                             \
+ product(bool, EnableAllLargePageSizes, false, EXPERIMENTAL,                \
+          "Client 11+ and Server 22+ only")                                 \
```

Comment:
**Windows** Client 11+ and ...  **Windows** Server...? Actually this description doesn't say what the flag is for.

Reply:
done